lib/events/events_handler.dart

```diff
@@ -19,13 +19,12 @@ class EventsHandler {
     try {
       handleDecodedEvent(json.decode(event));
     } on FormatException catch(e) {
-      print("Error while decoding event : $e");
+      print("Error while decoding event $event : $e");
```

Comment:
there is no space before ':' in english
you could double them if you want
print("Error while decoding event $event: $e");
or
print("Error while decoding event $event :: $e");

Reply:
That's right, fixed